hkpy/hkbase/hkrepository.py

```diff
@@ -242,7 +242,15 @@ def hyql(self, query: str, transitivity: Optional[bool] = False) -> HKEntityResu
         for entry in data:
             if isinstance(entry, dict):
                 entry = [entry]
-            row = [hkfy(e) for e in entry]
+            if isinstance(entry, list):
+                row = []
+                for e in entry:
+                    if isinstance(e, dict):
+                        row.append(hkfy(e))
+                    else:
+                        row.append(e)
+            else:
+                row = entry
```

Comment:
Is the `cast` in line 240 still valid? It makes seem like there is only two options, list of list of dicts or dicts. I'm gathering that maybe there are HyQL that return a single column with literals. It that it?

Reply:
It that's the case, I'd suggest adding these to the the Union in line 240.

Reply:
Fixed in: https://github.com/ibm-hyperknowledge/hkpy/commit/7bc08234892f20863c8934256af6a1d8e67a6734